installers/custom_ipxe/main_test.go

```diff
@@ -130,6 +130,25 @@ func TestIpxeScript(t *testing.T) {
 			echo userdata script
 			`,
 		},
+		{
+			"installer: ipxe data url",
+			"custom_ipxe",
+			&packet.InstallerData{Chain: "data:text/plain;charset=utf-8;base64,ZWNobyBteSBkYXRhIHVyaSBzY3JpcHQ="},
```

Comment:
this data uri is: `echo my data uri script`

Reply:
Since this comment reduced the mystery for me, can you add this as a comment in the code base?